frontend/src/components/place/PlaceVisitedPreview.vue

```diff
@@ -4,25 +4,55 @@
             <div class="entry-media">
                 <img 
                     class="image" 
-                    :src="visitedPlace.place_photo.url"
+                    src="http://www.fraufluger.ru/files/images/story/a7ab21562ce4476d11936f0fd3c27fa6_560x416.jpg"
```

Comment:
why did you do this?) You can download placeholder, put it to the assets directory, put path to the placeholder into store and it will work fine